packages/core/src/utilities/VoxelValue.ts

```diff
@@ -0,0 +1,235 @@
+import type { BoundsIJK, Point3, VolumeScalarData } from '../types';
+
+/**
+ * This is a simple, standard interface to values associated with a voxel.
+ */
```

Comment:
please add documentation about how the `new` it, what is get and what is set etc.